shapes/Shape.hpp

```diff
@@ -45,6 +45,7 @@ class Shape
     /* Shape Data */
     QPoint position;
 
+    virtual void applyProperties(QPainter&) const = 0;
```

Comment:
functions needs to breathe
```suggestion
    virtual void applyProperties(QPainter&) const = 0;
    
```